source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -4513,34 +4526,57 @@ export class Media extends CardElement {
                 this._mediaElement.appendChild(src);
             });
             element.appendChild(this._mediaElement);
-
-            var play_button = this.makeButton("▶️");
-            play_button.onclick = e => this.play();
-            element.appendChild(play_button);
-
-            var pause_button = this.makeButton("⏸️");
-            pause_button.onclick = e => this.pause();
-            element.appendChild(pause_button);
-
-            var stop_button = this.makeButton("⏹️");
-            stop_button.onclick = e => this.stop();
-            element.appendChild(stop_button);
-        }
+			
+			let controlBar = <HTMLDivElement>document.createElement("div");
+			controlBar.className = this.hostConfig.makeCssClassName("ac-controlbar");
+			controlBar.style.whiteSpace = "nowrap";
+
+            this._playButton = Media.makeButton("▶️", "play", ()=>this.play());
+            controlBar.appendChild(this._playButton);
+
+            this._pauseButton = Media.makeButton("⏸️", "pause", ()=>this.pause());
+			this._pauseButton.style.display = "none";
+            controlBar.appendChild(this._pauseButton);
+
+            this._resetButton = Media.makeButton("⏮️", "reset", ()=>this.reset());
+            controlBar.appendChild(this._resetButton);
+			
+			element.appendChild(controlBar);
+        }
+		else {
+			let img = new Image();
+			img.url = this.poster;
+			if (this.sources.length > 0) {
+				let openAction = new OpenUrlAction();
+				openAction.url = this.sources[0].url;
+				img.selectAction = openAction;
+			}
+			let renderedImage = img.render();
```

Comment:
Empty line after closing }

Reply:
I need to get these vertical whitespaces right. Doesn't come naturally, sorry.